tracker/api-client/Cargo.toml

```
@@ -0,0 +1,27 @@
+[package]
+name                 = "kallax-tracker-api-client"
+description          = "Tracker API client"
+version.workspace    = true
+authors.workspace    = true
+homepage.workspace   = true
+readme.workspace     = true
+repository.workspace = true
+license.workspace    = true
+edition.workspace    = true
+categories.workspace = true
+keywords.workspace   = true
+
+[dependencies]
+async-trait = "0.1"
+
+reqwest = { version = "0.11", features = ["json"] }
```

```suggestion
reqwest = { version = "0.11", features = ["json", "rustls"] }
```
Use `rustls` instead of other `tls` options.

---

wtf rustls

---

`rustls` solves the issue https://github.com/thxnet/kallax/actions/runs/5847955236/job/15854731415?pr=120

---

cool